packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -27,18 +31,29 @@ import {partitionList} from './utils';
 import {diag} from '@opentelemetry/api';
 import {mapOtelResourceToMonitoredResource} from '@google-cloud/opentelemetry-resource-util';
 
+import {VERSION} from './version';
+
 // Stackdriver Monitoring v3 only accepts up to 200 TimeSeries per
 // CreateTimeSeries call.
 const MAX_BATCH_EXPORT_SIZE = 200;
 
-const OT_USER_AGENT = {
-  product: 'opentelemetry-js',
-  version: VERSION,
-};
+const OT_USER_AGENTS = [
+  {
+    product: 'opentelemetry-js',
+    version: OT_VERSION,
+  },
+  {
+    product: 'google-cloud-metric-exporter',
+    version: VERSION,
+  },
+];
 const OT_REQUEST_HEADER = {
   'x-opentelemetry-outgoing-request': 0x1,
 };
-google.options({headers: OT_REQUEST_HEADER});
+google.options({
+  headers: OT_REQUEST_HEADER,
+  userAgentDirectives: OT_USER_AGENTS,
```

Comment:
This yields a slightly different format than we use in other exporters (see [above](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/451/files#diff-eff067595506c23393cabc321341b3a854a603444c67a52eff82e593be38264d)), b/c `googleapis-common` accepts structured user agent "directive".

I can override the WHOLE header but then we lose the `google-api-nodejs-client/5.1.0 (gzip)` part. Does anyone prefer this approach?